lib/routes.js

```diff
@@ -9,6 +9,14 @@ module.exports = {
     inputs: ['query', 'cookies'],
     sinks: sinks.cmdInjection
   },
+  cmdInjectionSemanticChainedCommands: {
+    base: '/cmdInjectionSemanticChainedCommands',
+    name: 'Command Injection Semantic Chained Commands',
+    link: 'https://www.owasp.org/index.php/Command_Injection',
```

Comment:
not sure what to do for the link here, semantic chained commands is more specific than cmd injection.  also it's different we block if there's a chained command _regardless_ of whether or not it came from a user input (it may or may not have).

Reply:
I think this is probably fine just to link to the generic rule documentation. :man_shrugging:

Reply:
Yea agreed, just calling it out